dev/AppNotifications/AppNotificationManager.cpp

```diff
@@ -14,6 +14,7 @@
 #include "NotificationTransientProperties.h"
 #include "AppNotification.h"
 #include <NotificationProgressData.h>
+#include "AppNotificationTelemetry.h"
```

Comment:
Please do not baseline against the L1. The L1 is strictly for PR comments and P0 work.

Reply:
I would baseline this against Pauls bug fix branch which has the latest changes

Reply:
It's the best base branch to use for my work and I want to keep the option of merging directly into the L1 open in case it slips since this work needs to be in for Friday. Either the branch will merge early as planned and I'll re-target my work to main, or it will slip and I'll merge directly into the L1